practical-unit-testing-with-rtl/Exercise2/solution/src/components/Board/Board.tsx

```diff
@@ -0,0 +1,34 @@
+import Square from "./Square";
+import { useTicTacToe } from "../../hooks/useTicTacToe";
+import GameInformation from "./GameInformation";
+
+import styles from "./Board.module.css";
+
+const Board = () => {
+  const { board, handleSquareSelect, reset, currentTurn, winner, isGameOver } =
+    useTicTacToe();
+
+  return (
+    <div>
+      <GameInformation
+        isGameOver={isGameOver}
+        winner={winner}
+        currentTurn={currentTurn}
+        reset={reset}
+      />
+      <div data-testid="board" className={styles.board}>
+        {board.map((value, squareIndex) => (
+          <Square
+            key={squareIndex}
+            value={value}
+            onSquareSelect={() => {
+              !winner && handleSquareSelect(squareIndex);
+            }}
+          />
+        ))}
+      </div>
```

Comment:
I’m fairly certain this would be more accessible as a `table` because it comes with column & row semantics, which would make it easier to move around by default.